arch/arm/Kconfig

```diff
@@ -14,6 +14,203 @@ config CPU_CORTEX
 	help
 	  This option signifies the use of a CPU of the Cortex family.
 
+# Math coprocessor symbols; these should be selected by the CPU symbol to
```

Comment:
If it is better to move these VFP configs into a new Kconfig file? like Kconfig.vfp or something else

Reply:
fixed